aspnetcore/migration/fx-to-core/areas/authentication.md

```diff
@@ -0,0 +1,374 @@
+---
+title: ASP.NET Framework to Core Authentication Migration
+description: ASP.NET Framework to Core Authentication Migration
+author: rick-anderson
+ms.author: riande
+monikerRange: '>= aspnetcore-6.0'
+ms.date: 11/9/2022
```

Comment:
If you consider this topic still accurate and therefore up to date, we might change the ms.date to the current date so folks will see it is considered up to date.  Otherwise, out of scope for this PR and just leave as is.